cmd/config.go

```diff
@@ -0,0 +1,66 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/lunarway/shuttle/pkg/git"
+	"github.com/lunarway/shuttle/pkg/ui"
+	"github.com/spf13/cobra"
+)
+
+func newConfig(uii *ui.UI, contextProvder contextProvider) *cobra.Command {
+	var envVarsToExclude []string
+
+	envCmd := &cobra.Command{
+		Use:   "config",
+		Short: "Display shuttle context information",
+		Run: func(cmd *cobra.Command, args []string) {
+			uii.SetContext(ui.LevelSilent)
+			environmentVariables := os.Environ()
+			shouldExclude := make(map[string]bool)
+
+			for _, envVarToExclude := range envVarsToExclude {
+				shouldExclude[envVarToExclude] = true
+			}
+
+			context, err := contextProvder()
+			if err != nil {
+				panic(err)
```

Comment:
By using the `RunE` field instead of `Run` we can return an error to `cobra`. That way the command line will do error handling instead of this panic which will just terminate the command and dump a stacktrace.

```go
cobra.Command{
  RunE: func(cmd *cobra.Command, args []string) error {
  }
}
```

```suggestion
				return err
```